jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/jdbc/SchemaAdapterNotesJsonConverter.java

```diff
@@ -0,0 +1,110 @@
+package com.exasol.adapter.jdbc;
+
+import com.exasol.adapter.AdapterException;
+import com.exasol.utils.JsonHelper;
+
+import javax.json.JsonBuilderFactory;
+import javax.json.JsonObject;
+import javax.json.JsonObjectBuilder;
+
+/**
+ * Converts schema adapter Notes into JSON format and back
+ */
+public final class SchemaAdapterNotesJsonConverter {
+    private static final String CATALOG_SEPARATOR = "catalogSeparator";
+    private static final String IDENTIFIER_QUOTE_STRING = "identifierQuoteString";
+    private static final String STORES_LOWER_CASE_IDENTIFIERS = "storesLowerCaseIdentifiers";
+    private static final String STORES_UPPER_CASE_IDENTIFIERS = "storesUpperCaseIdentifiers";
+    private static final String STORES_MIXED_CASE_IDENTIFIERS = "storesMixedCaseIdentifiers";
+    private static final String SUPPORTS_MIXED_CASE_IDENTIFIERS = "supportsMixedCaseIdentifiers";
+    private static final String STORES_LOWER_CASE_QUOTED_IDENTIFIERS = "storesLowerCaseQuotedIdentifiers";
+    private static final String STORES_UPPER_CASE_QUOTED_IDENTIFIERS = "storesUpperCaseQuotedIdentifiers";
+    private static final String STORES_MIXED_CASE_QUOTED_IDENTIFIERS = "storesMixedCaseQuotedIdentifiers";
+    private static final String SUPPORTS_MIXED_CASE_QUOTED_IDENTIFIERS = "supportsMixedCaseQuotedIdentifiers";
+    private static final String NULLS_ARE_SORTED_AT_END = "nullsAreSortedAtEnd";
+    private static final String NULLS_ARE_SORTED_AT_START = "nullsAreSortedAtStart";
+    private static final String NULLS_ARE_SORTED_HIGH = "nullsAreSortedHigh";
+    private static final String NULLS_ARE_SORTED_LOW = "nullsAreSortedLow";
+
+    public static SchemaAdapterNotesJsonConverter getInstance() {
+        return new SchemaAdapterNotesJsonConverter();
+    }
+
+    private SchemaAdapterNotesJsonConverter(){
+    }
```

Comment:
Add comment to show that this is intentionally empty.

Reply:
fixed